cypress/features/regression/dialogFullScreen.feature

```diff
@@ -7,7 +7,6 @@ Feature: Dialog Full Screen component
   @positive
   Scenario: CloseIcon has the border outline
     Given I open component preview
```

Comment:
Should be always `Given` - `When` - `Then` of `When` - `Then`
```suggestion
    When I open component preview
```
We need to change `Given` to `When` here - because we removed `When` step